### PR TITLE
P1082R0 LWG Issues to be moved in Rapperswil (except issue 3071)

### LINKED ISSUE #2129: [2018-06 LWG Motion 11] P1082R0 LWG Issues to be moved in Rapperswil (except issue 3071)
Except issue 3071.

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #2129.

### CODE REVIEW

**Comment by tkoeppe** [comment]
Regarding P1082R0: Could you double-check that you've applied the edit to meta.trans.ptr?

File: source/iostreams.tex [hunk 1]
```diff
@@ -14800,21 +14800,18 @@
...
 \begin{itemdescr}
 \pnum
-\effects Establishes the postcondition by attempting to create the
+\effects Creates the
   directory \tcode{p} resolves to, with
   attributes copied from directory \tcode{existing_p}. The set of attributes
-  copied is operating system dependent. Creation failure because \tcode{p} resolves to an existing directory shall not be
-  treated as an error.
+  copied is operating system dependent.
+  Creation failure because \tcode{p} already exists is not treated as an error.
```
Comment (tkoeppe) [inline source/iostreams.tex]: "treated as" should be deleted.

Comment (jensmaurer) [reply source/iostreams.tex]: Fixed.

File: source/utilities.tex [hunk 2]
```diff
@@ -19514,9 +19514,10 @@
 which are converted to arrays of \tcode{charT} using a decimal conversion with no leading zeroes.
 
 \pnum
-For streams where \tcode{charT} has an 8-bit representation,
-\tcode{"\textmu{}s"} should be encoded as UTF-8. Otherwise UTF-16 or UTF-32 is encouraged.
-The implementation may substitute other encodings, including \tcode{"us"}.
+If \tcode{Period::type} is \tcode{micro},
+but the character U+00B5 cannot be represented in
```
[the remark this answers is not in the document]
Comment (jensmaurer) [reply source/utilities.tex]: @zygoloid, any guidance here?

**Comment by jensmaurer** [comment]
@tkoeppe: I did not update meta.trans.ptr, because common_type is in meta.trans.other. Furthermore, the underlying text has changed quite a bit here, the salient parts are now in "Note A" and "Note B", below the table. I think I did the best I could given the incoming text, but feel free to make specific improvement suggestions.

File: source/utilities.tex (hunk 2 above)
[the remark this answers is not in the document]
Comment (zygoloid) [reply source/utilities.tex]: U+00B5 is really how the Unicode standard says things. See http://www.unicode.org/versions/Unicode11.0.0/appA.pdf

I see no reason to deviate from the relevant normative reference's notational convention here. Using U+00B5 MICRO SIGN (with MICRO SIGN in small-caps) seems OK too. I'm going to take this as-is for now.

**Comment by zygoloid** [comment]
I believe Thomas's comments have all been addressed, and I've done a review pass myself. LWG2139 is a bit of a merge conflict disaster, but I've convinced myself we've made all the requested changes.

**Pull request merged**